modules/Bio/EnsEMBL/Variation/AnnotationFile.pm

```diff
@@ -0,0 +1,175 @@
+=head1 LICENSE
```

Comment:
As this module is not used on its own (serve as base for `GERPFile` and `CADDFile`, maybe we could rename it `BaseAnnotationFile`